### DISCUSSION TITLE
Oqtane 6.0.0

### DISCUSSION THREAD

**Discussion by SSFlyer01** [oqtane/Oqtane.Theme.Corporate#7] [Q&A]
Is this compatible with Oqtane 6.0.0? It all compiled well, after I changed the target framework to 9,, but the theme doesn't show up when running the site and trying to select it.
It did copy the theme to the oqtane.server wwwroot\theme directory.
Thanks!

**Comment by sbwalker** [comment]
Yes it is compatible. You would need to change the net8.0 references to net9.0 in the Debug.cmd in the Package project.

**Comment by sbwalker** [comment]
@SSFlyer01 it has been upgraded to .NET 9 and Oqtane 6.1
